metadata_writer_inventory_conn.c

```diff
@@ -334,15 +337,13 @@ static void md_inventory_insert_fake_mode(struct md_writer_sqlite *mws,
                                        struct md_conn_event *mce,
                                        uint8_t mode)
 {
-    const char *event_value_str = mce->event_value_str;
     int32_t retval;
 
     //TODO: Not a nice way to access parent variable
     mce->sequence = mde_inc_seq(mws->parent);
     mce->event_value = mode;
     mce->event_param = CONN_EVENT_META_MODE_UPDATE;
-    mce->event_value_str = NULL;
```

Comment:
Don't we need this one to create SQL statements that are compatible with the old database structure?